src/java/org/apache/cassandra/service/disk/usage/DiskUsageState.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.service.disk.usage;
+
+import org.apache.cassandra.db.guardrails.GuardrailsConfig;
+
+public enum DiskUsageState
+{
+    /** Either disk usage guardrail is not enabled or gossip state is not ready. */
+    NOT_AVAILABLE("Not Available"),
+
+    /**
+     * Disk usage is below both {@link GuardrailsConfig#getCollectionSizeWarnThreshold()} and
+     * {@link GuardrailsConfig#getCollectionSizeFailThreshold()}.
```

Comment:
I started working on the 0 patch and noticed this java doc, GuardrailsConfig#getCollectionSizeWarnThreshold(), I think this was copy-paste mistake, also the other JavaDocs in this class. Apologize for not catching it earlier :(

Reply:
Good catch, I should have been thinking on those 0 values for collection sizes when using autocompletion :). Just fixed it.